content/300-guides/100-performance-and-optimization/150-connection-management/index.mdx

```diff
@@ -89,25 +89,21 @@ As a workaround, you can store `PrismaClient` as a global variable in developmen
 ```ts file=client.ts
 import { PrismaClient } from '@prisma/client'
 
-// add prisma to the NodeJS global type
-interface CustomNodeJsGlobal extends NodeJS.Global {
-  prisma: PrismaClient
-}
-
-// Prevent multiple instances of Prisma Client in development
-declare const global: CustomNodeJsGlobal
-
-const prisma = global.prisma || new PrismaClient()
+const globalForPrisma = global as unknown as { prisma: PrismaClient }
 
-if (process.env.NODE_ENV === 'development') global.prisma = prisma
+export const prisma =
+  globalForPrisma.prisma ||
+  new PrismaClient({
+    log: ['query'],
+  })
```

Comment:
Curious why logging was added here? 

Logging wasn't mentioned in the previous version of this snippet. If it was added for a specific reason, is it worth pointing out that reasoning explicitly?

Reply:
There isn't any specific reason for adding logging. I have reused the code snippet that was used in this Help Article: https://www.prisma.io/docs/guides/database/troubleshooting-orm/help-articles/nextjs-prisma-client-dev-practices

In the Help Article, logging was enabled, so I did the same here.
I agree that it doesn't add any value. I'll remove it.

Reply:
Removed Logging 👍